hpx/runtime/threads/detail/scheduled_thread_pool_impl.hpp

```diff
@@ -134,8 +133,11 @@ namespace hpx { namespace threads { namespace detail
            << "\n"
            << "is running on PUs : \n";
         os << std::hex << HPX_CPU_MASK_PREFIX << get_used_processing_units()
+           << " "
+           << std::bitset<HPX_HAVE_MAX_CPU_COUNT>(get_used_processing_units())
```

Comment:
What's the rationale not to use the existing `mask_type` for this?

Reply:
on machines where `HPX_HAVE_MAX_CPU_COUNT` is less than 64 bits, the output is just an uint, I wanted to see a binary bit mask explicitly - hence the cast to std::bitset on cout. (debugging).